api/api.go

```diff
@@ -8,8 +8,8 @@ import (
 type ClientService interface {
 	ListServices(inst flux.InstanceID, namespace string) ([]flux.ServiceStatus, error)
 	ListImages(flux.InstanceID, flux.ServiceSpec) ([]flux.ImageStatus, error)
-	PostRelease(flux.InstanceID, flux.ReleaseJobSpec) (flux.ReleaseID, error)
-	GetRelease(flux.InstanceID, flux.ReleaseID) (flux.ReleaseJob, error)
+	PostRelease(flux.InstanceID, flux.ReleaseJobParams) (flux.JobID, error)
+	GetRelease(flux.InstanceID, flux.JobID) (flux.Job, error)
```

Comment:
For the record, the difficulty with bumping the version, without supporting the old version of the API, is that you have to use a different client binary to talk to different running services, e.g., local v dev v prod, until the change has made it through to all of them.